.planning/sketches/themes/default.css

```
@@ -0,0 +1,32 @@
+  --color-bg: #09090b;
+  --color-surface: #12121a;
+  --color-surface-elevated: #1a1a24;
+  --color-border: #3a3a4a;
+  --color-text: #fafafa;
+  --color-text-muted: #94969c;
+  --color-primary: #a21caf;
+  --color-primary-hover: #c026d3;
+  --color-accent: #00d4ff;
+  --color-success: #10b981;
+  --color-danger: #ef4444;
+  --font-sans: Inter, ui-sans-serif, system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", sans-serif;
```

Stylelint (17.14.0)

[error] Expected "Inter" to be "inter" (value-keyword-case)

(value-keyword-case)